src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -104,6 +105,8 @@ public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, Expor
 
 
     form.onExport(() -> backgroundExecutor.execute(() -> {
+      //notify user that export started
```

Comment:
Ugh, sorry! This comment slipped too. I think the code is self-explanatory. Remove comment :)